website/routes.py

```diff
@@ -1166,6 +1166,16 @@ def make_url_map(app):
             json_renderer,
         ),
 
+        Rule(
+            [
+                '/project/<pid>/contributor/remove/',
+                '/project/<pid>/node/<nid>/contributor/remove/',
```

Comment:
Looks like the project and the node needn't even be related. If I make a new project, I can get any other project or component at:

/project/<new-unrelated-project-id>/node/<any-node-id>/contributors/

`<pid>` must be an actual project id, but it needn't be one that I have direct access to in order to use `node/` URLs under it.